exchange_machine.lua

```diff
@@ -87,16 +77,18 @@ local tablecolumns =
 
 
 local function table_from_results(results, x, y, w, h, selected)
-	local fs_tab = {}
+	local fs_tab
 
 	local function insert(str)
-		table.insert(fs_tab, str)
+		fs_tab[#fs_tab+1] = str
```

Comment:
My understanding is that table.insert basically just does an assignment to index #tab + 1. Is this just to avoid the overhead of a function call (and table access, since I didn't localize table.insert)?